docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -86,7 +86,7 @@ You can make sure which version is used by the interpreter by running this check
 .. code-block:: bash
 
     root@b8a8e73caa2c:/opt/airflow# python
-    Python 3.6.12 (default, Nov 25 2020, 03:59:00)
+    Python 3.7.12 (default, Nov 25 2020, 03:59:00)
     [GCC 8.3.0] on linux
     Type "help", "copyright", "credits" or "license" for more information.
```

Comment:
```suggestion
```

This one shouldn't be changed. Maybe we just rip out all this since it isn't really important?